site/content/admin-guide/amqp-adapter-config.md

```diff
@@ -10,6 +10,8 @@ The adapter is implemented as a Spring Boot application. It can be run either di
 
 ## Service Configuration
 
+In addition the following options, this component supports the options described in [Common Configuration]({{< relref "common-config.md" >}}).
```

Comment:
IMHO this should read
`In addition to the following ...`

Reply:
Indeed. Should be fixed.